mods/ra/maps/soviet-05/main.lua

```diff
@@ -0,0 +1,212 @@
+	Actor.Create("camera", true, { Owner = player, Location = SovietBasePoint.Location })
+
+	startmcv.Move(MCVStartMovePoint.Location)
+	Runner1.Move(RunnerRetreatPoint.Location)
+	Runner2.Move(RunnerRetreatPoint.Location)
+	Runner3.Move(RunnerRetreatPoint.Location)
+
+	ProduceInfantry()
+	Trigger.AfterDelay(DateTime.Minutes(2), ProduceShips)
+
+	Media.PlaySpeechNotification(player, "ReinforcementsArrived")
+
+	if Map.Difficulty == "Hard" or Map.Difficulty == "Medium" then
+		Trigger.AfterDelay(DateTime.Seconds(25), Gguy1)
+	end
+	Trigger.AfterDelay(DateTime.Minutes(2), Gguy1)
+	Trigger.AfterDelay(DateTime.Minutes(5), Gguy1)
+end
+
+Expand = function()
+	if not ExpansionCheck then
+		if not mcvtransport.IsDead then
+		mcvtransport.Move(mcvGGLoadPoint.Location)
+		end
+		if not mcvGG.IsDead then
+		mcvGG.Move(mcvGGLoadPoint.Location)
+		end
+		Reinforcements.Reinforce(GoodGuy, {"dd", "dd"}, {ReinfNorthPoint.Location, NearDockPoint.Location}, 0, function(ddsquad)
+			ddsquad.AttackMove(NearExpansionPoint.Location)
+		end)
+		ExpansionCheck = true
+		Trigger.ClearAll(mcvGG)
+		Trigger.ClearAll(mcvtransport)
+		Trigger.AfterDelay(DateTime.Seconds(3), function(load)
+			if not mcvtransport.IsDead then
+			mcvGG.IsInWorld = false
+			else
+				return
+			end
+			if not mcvtransport.IsDead then
+				mcvtransport.LoadPassenger(mcvGG)
+				mcvtransport.Move(mcvGGUnloadPoint.Location)
+			else
+				return
+			end
+			Trigger.AfterDelay(DateTime.Seconds(10), function(unload)
+				if not mcvtransport.IsDead then
+					mcvtransport.UnloadPassengers()
+				end
+				Trigger.AfterDelay(DateTime.Seconds(2), function(move)
+					if not mcvGG.IsDead then
+					mcvGG.Move(MCVDeploy.Location)
+					end
+					Trigger.AfterDelay(DateTime.Seconds(4), function(deploy)
+						if not mcvGG.IsDead then
+						mcvGG.Deploy()
+						end
+						if not mcvtransport.IsDead then
+							mcvtransport.Move(ReinfNorthPoint.Location)
+						end
+						IslandTroops()
+						Trigger.AfterDelay(DateTime.Seconds(7), function(leave)
+							if not mcvtransport.IsDead then
+								mcvtransport.Destroy()
+							end
+							BuildBase()
+						end)
+					end)
+				end)
+			end)
+		end)
+	else
+	end
+end
+
+Tick = function()
+	if Greece.HasNoRequiredUnits() and GoodGuy.HasNoRequiredUnits() then
+		player.MarkCompletedObjective(KillAll)
+	end
+
+	if player.HasNoRequiredUnits() then
+		player.MarkFailedObjective(CaptureObjective)
+		player.MarkFailedObjective(KillAll)
+		GoodGuy.MarkCompletedObjective(BeatUSSR)
+	end
+
+	if Radar.IsDead then
+		player.MarkFailedObjective(CaptureObjective)
+	end
+
+	if Radar.Owner == player and not Radstop then
+		if not ExpansionCheck then
+			Expand()
+			ExpansionCheck = true
+		end
+
+		player.MarkCompletedObjective(CaptureObjective)
+		Reinforcements.Reinforce(Greece, gguy2, {CrossroadPoint.Location, GreeceBaseEntryPoint.Location, GreeceBasePoint.Location}, 0, function(soldier)
```

Comment:
Also missing spaces before/after the brackets. It should be:
`{ CrossroadPoint.Location, GreeceBaseEntryPoint.Location, GreeceBasePoint.Location },`